zstacklib/zstacklib/utils/gpu.py

```
@@ -496,11 +496,14 @@ def is_valid_co_processor(description, vendor_id):
 
 
 def get_vastai_type():
-    r, o, e = bash_roe("lspci | grep Vastai")
+    r, o, e = bash_roe("lspci | grep -E 'Vastai|1ec6'")
```

Ruff (0.12.2)

`bash_roe` may be undefined, or defined from star imports

(F405)